test/controllers/api/stats_controller_test.rb

```diff
@@ -34,6 +34,19 @@ class Api::StatsControllerTest < ActionDispatch::IntegrationTest
       [1.days.ago.to_date.to_s, 1],
       [0.days.ago.to_date.to_s, 23]
     ]
+    get "/api/stats/channel_and_email_verified_signups_per_day", headers: { "HTTP_AUTHORIZATION" => "Token token=fake_api_auth_token" }
+
+    assert_equal 200, response.status
+    resp = JSON.parse(response.body)
+    assert_equal resp, [
+      [6.days.ago.to_date.to_s, 1],
+      [5.days.ago.to_date.to_s, 0],
+      [4.days.ago.to_date.to_s, 0],
+      [3.days.ago.to_date.to_s, 0],
+      [2.days.ago.to_date.to_s, 0],
+      [1.days.ago.to_date.to_s, 1],
+      [0.days.ago.to_date.to_s, 14]
```

Comment:
Is there some way to get the number of sign ups dynamically so we don't need to update this test every time a fixture is added?

Reply:
Sure thing, I'll refactor.